packages/react-components/react-carousel-preview/library/src/components/CarouselCard/useCarouselCard.ts

```diff
@@ -14,64 +15,54 @@ import { useCarouselStore_unstable } from '../useCarouselStore';
  * @param props - props from this instance of CarouselCard
  * @param ref - reference to root HTMLDivElement of CarouselCard
  */
+
 export const useCarouselCard_unstable = (
   props: CarouselCardProps,
   ref: React.Ref<HTMLDivElement>,
 ): CarouselCardState => {
   const { value } = props;
-  const { circular, peeking } = useCarouselContext_unstable();
 
-  const visible = useCarouselStore_unstable(snapshot => snapshot.activeValue === value);
-  const peekDir: 'prev' | 'next' | undefined = useCarouselStore_unstable(snapshot => {
-    if (!peeking) {
-      return;
-    }
+  const elementRef = React.useRef<HTMLDivElement>(null);
+  const isActiveIndex = useCarouselStore_unstable(snapshot => snapshot.activeValue === value);
+
+  React.useEffect(() => {
+    const element = elementRef.current;
 
-    const currentIndex = snapshot.activeValue ? snapshot.values.indexOf(snapshot.activeValue) : null;
+    if (element) {
+      const listener = (_e: Event) => {
+        const event = _e as CustomEvent<{ isVisible: boolean }>;
+        const hidden = !event.detail.isVisible;
 
-    if (currentIndex !== null && currentIndex >= 0) {
-      let nextValue = currentIndex + 1 < snapshot.values.length ? snapshot.values[currentIndex + 1] : null;
-      let prevValue = currentIndex - 1 >= 0 ? snapshot.values[currentIndex - 1] : null;
+        element.ariaHidden = hidden.toString();
+        element.inert = hidden;
```

Comment:
why is `inert` being used here? Is the expectation that users shouldn't be able to focus the card?

Reply:
Resolving as part of groups and focus features PR